### ISSUE TITLE
ECONNRESET

### ISSUE THREAD

**Issue by unixweb** [nabbl/node-red-contrib-google-home-notify#10]
Sometime hangs the communication with this error and only solution is to restart node-red.

{ payload: 'Der aktuelle Luftdruck 945 Hecto Pascal.\r',
  _msgid: 'ae65c197.9679b' }
Error: read ECONNRESET
{ topic: 'toggle', payload: 'toggle', _msgid: '97ea7c2d.09e39' }
{ topic: 'toggle', payload: 'toggle', _msgid: '97ea7c2d.09e39' }
{ payload: 'Der aktuelle Luftdruck 946 Hecto Pascal.\r',
  _msgid: '60901c7c.ddd754' }
Error: read ECONNRESET
Error: read ECONNRESET

**Comment by Dilepa** [comment]
Any solutions

**Comment by mikebiddell** [comment]
And me. At least once a day. Socket hang up. Only fix is a reboot.

**Comment by jenskastensson** [comment]
Yes, it's a pity because it is such a nice and useful node but not
reliable.

**Comment by mikebiddell** [comment]
I've been waiting for this node for a couple of years...... very useful. Just needs stability

**Comment by johnty** [comment]
no updates? experiencing the same behaviour. a reboot of node-red appears to fix it. (redeploying doesn't help)

**Comment by tonyadderley** [comment]
Hi - I am experiencing this issue - I wondered if anyone had any suggestions of a workaround , ideally that can be scripted to "autoreset" ?

**Comment by howsTricks** [comment]
Also experiencing this. Anything that can be done to troubleshoot? I am using google home minis, both experience this issue.

**Comment by kodarn** [comment]
Also seen this; a reboot of the Google Home (mini) seems to work just as a reboot of the NodeRed instance.

**Comment by benkap** [comment]
Same issue here.... any fix in the roadmap?
